### PR TITLE
Fix duplicate queries + tools for tracing

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: common/src/main/scala/explore/components/graphql/LiveQueryRenderMod.scala [hunk 1]
```diff
@@ -89,67 +86,6 @@ object LiveQueryRenderMod {
             State.apply
           )
       )
-      .componentDidMount { $ =>
```
Comment (rpiaggio) [inline common/src/main/scala/explore/components/graphql/LiveQueryRenderMod.scala:92]: All this was abstracted away to `Render` in the last refactor, but I forgot to remove it from here. This was causing duplicate queries.

File: targeteditor/src/main/scala/explore/targeteditor/TargetBody.scala [hunk 2]
```diff
@@ -54,14 +52,10 @@ object TargetBody {
 
   implicit val propsReuse = Reusability.derive[Props]
 
-  class Backend($ : BackendScope[Props, Unit]) {
+  class Backend {
     // Create a mutable reference
     private val aladinRef = Ref.toScalaComponent(AladinRef)
```
Comment (cquiroz) [inline targeteditor/src/main/scala/explore/targeteditor/TargetBody.scala:60]: Odd we don't need this anymore

**Review by cquiroz — APPROVED** [review]
LGTM

File: targeteditor/src/main/scala/explore/targeteditor/TargetBody.scala (hunk 2 above)
Comment (rpiaggio) [reply targeteditor/src/main/scala/explore/targeteditor/TargetBody.scala:60]: It was only being used in `setName`, which isn't used anymore.

**Pull request merged**